### ISSUE TITLE
Importing app does not preserve Navigation

### ISSUE THREAD

**Issue by B-z-F** [Budibase/budibase#7152]
**Hosting**
- Self
  - Method: docker compose
  - Budibase Version: 1.1.17

**Describe the bug**
When I create an app, modify Navigation (Position: Vertical; Logo: off; custom Background and Text color; add some links) then after export and "Create new app" -> "Import app" the new app has default Navigation (Position: Horizontal; Logo: on; default Background and Text color; no links)

**Comment by aderici** [comment]
Yeah, i can confirm after having to restore a backup into a new system. I hope thats the only thing that will be lost
